src/styles/blocks/header.scss

```diff
@@ -38,5 +38,11 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+
+    transition: color $transition-delay;
```

Comment:
Here the same mistake (replace `color` to real color #....)

Reply:
This is the same situation. I applied transition to color in nav block